src/refdb_fs.c

```diff
@@ -43,6 +43,14 @@ enum {
 	PEELING_FULL
 };
 
+typedef enum {
+	GIT_REFDB__FORCE_WRITE = (1 << 0),
+	GIT_REFDB__SKIP_REFLOG = (1 << 1),
```

Comment:
I think these two flags should be documented, as well. `SKIP_REFLOG` is obvious, but I'm left to guess with regards to the first one.